draft-ietf-mls-protocol.md

```diff
@@ -610,7 +614,7 @@ In this tree, we can see all three of the above rules in play:
 ## Ratchet Tree Updates
 
 In order to update the state of the group such as adding and
-removing clients, MLS messages are used to make changes to the
+removing clients, Handshake messages are used to make changes to the
```

Comment:
In general, I've been using lower-case "handshake message", since we don't have a Handshake struct any more.